Assignment one

abeerkhe opened the pull request:
<!-- PULL REQUEST TITLE: UofT-DSI | <Module Name> - Assignment <assignment number>-->

## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)

Upload my completed Assignment 1 for the SQL module.

## What did you learn from the changes you have made?

How to query tables, join tables, create temp tables, etc. - basic SQL functionality.

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?

N/A

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?

The main challenges were recalling to add data to temp tables and using the STRFTIME function which we didn't cover in class yet. I reviewed class material and used Google to help cover knowledge gaps and trial and error in SQLite to figure out the right syntax and approaches.

## How were these changes tested?

I ran SQL queries a number of times in SQLite.

## A reference to a related issue in your repository (if applicable)
N/A

## Checklist
- [ x] I can confirm that my changes are working as intended

abeerkhe committed:
Completed SQL Assignment 1 for submission.

abeerkhe committed:
Added my Assignment 1 data model.

monzchan commented on 02_activities/assignments/assignment1.sql:
```diff
@@ -39,19 +50,43 @@ Using the product table, write a query that outputs the product_id and product_n
+	WHEN 'unit' 
+		THEN 'unit' 
+	ELSE 'bulk'
+END AS product_qty_type_condesed
```
typo here 'condesed'

abeerkhe replied:
Updated.

monzchan commented on 02_activities/assignments/assignment1.sql:
```diff
@@ -26,8 +33,12 @@ filtered by vendor IDs between 8 and 10 (inclusive) using either:
 
+SELECT *, (quantity * cost_to_customer_per_qty) AS price 
+FROM customer_purchases 
+WHERE vendor_id BETWEEN 4 AND 10
```
The filter should be between 8 and 10

abeerkhe replied:
Updated.

monzchan reviewed (changes requested):
Almost there! :)

monzchan reviewed (changes requested):
Please see the comments :)

abeerkhe committed:
Making updates to SQL Assignment 1 as per grader feedback. ß

monzchan reviewed (approved):
Excellent work!!

abeerkhe committed:
Submission for Assignment 2. Late Submission
was confirmed with Thomas in advance as I was
sick last week and over the weekend.
